docs/db/migrations.md

```diff
@@ -154,6 +154,44 @@ type Note {
 
 This maps to `comment.note_id` in the database.
 
+##### ManyToMany
+
+To create a many-to-many relationship, add a model for your join table and use two one-to-many relationship mappings to create the relationship.
+
+```graphql
+""" 
+@model 
+"""
+type Note {
+  id: ID!
+  title: String!
+  description: String
+  """
+  @oneToMany field: 'note'
+  """
+  authors: [NoteAuthor]
+}
+
+"""
+@model
+"""
+type NoteAuthor {
+  id: ID!
+}
+
+"""
+@model
+"""
+type User {
```

Comment:
While I do see this working on the db level. It is hard for me to get how this can be queried with Graphback now.

Reply:
```
getUser(...){
 notes: {
   note: {
      ??
   }
 }
}
```

I need to test this in production app we have - there are couple examples where this relation makes sense. There is also problem with lack of client side generation capabilities on the client to support 3 levels of relations now